requests-for-comments/2022-09-08-notifications.md

```diff
@@ -0,0 +1,81 @@
+# Request for comments: Notifications & @pauldambra 
+
+## Problem statement
+*Who are we building for, what are their needs (include example cases), why is this important?*
+
+Users don't know when to come back to PostHog. Something important can happen and they might miss it. This can be important things in the data, or that another user has edited your insight.
```

Comment:
> that another user has edited your insight.

Q: Is it important to know about this when it happens? What do notifs add here over the audit log?

Reply:
They add timeliness. The audit log is passive - you have to go check it to find new information. Notifications can push important information directly to the user regardless of their context in the product. Additionally notifications can span multiple contexts, so they can communicate a broader range of information in very specific contexts. Audit logs are often specific to a single feature, so they only cover that scope.